src/hotspot/share/utilities/ostream.hpp

```diff
@@ -29,6 +29,7 @@
 #include "runtime/timer.hpp"
 #include "utilities/globalDefinitions.hpp"
 #include "utilities/macros.hpp"
+#include <cstdint>
```

Comment:
This std header doesn't seem to be used anywhere else in hotspot code - why do we need it here?

Reply:
Good catch, it's been removed.